ros2topic/test/test_echo_pub.py

```diff
@@ -245,3 +245,43 @@ def publish_message():
                     # Cleanup
                     self.node.destroy_timer(publish_timer)
                     self.node.destroy_publisher(publisher)
+
+    @launch_testing.markers.retry_on_failure(times=5)
+    def test_echo_raw(self, launch_service, proc_info, proc_output):
+        topic = '/clitest/topic/echo_raw'
+        publisher = self.node.create_publisher(String, topic, 10)
+        assert publisher
+
+        def publish_message():
+            publisher.publish(String(data='hello'))
+
+        publish_timer = self.node.create_timer(0.5, publish_message)
+
+        try:
+            command_action = ExecuteProcess(
+                cmd=(['ros2', 'topic', 'echo', '--raw', topic, 'std_msgs/msg/String']),
```

Comment:
@jacobperron no need for the extra parenthesis.

Reply:
removed in 72d5e00